src/DssDirectDepositAaveDai.t.sol

```diff
@@ -29,33 +28,55 @@ interface Hevm {
     function load(address,bytes32) external view returns (bytes32);
 }
 
-interface AuthLike {
-    function wards(address) external returns (uint256);
+interface GemAbstract {
```

Comment:
Why not to call them `Like` everywhere? and maybe avoid the inheritance just duplicate the functions whenever is necessary.
using: `import({...} from '');` to import just the contract to test will avoid the issue of using the same naming for the interfaces.

Reply:
yeah, this approach was used in `vote-delegate`.
The current one make use of existing code interfaces expanding them were needed, to reduce code size, favouring reusability.
I see your point though, and I think it's more elegant overall.

Reply:
done in 06bd9e9